### PR TITLE
Add pre-commit

### PR DESCRIPTION
**Pull request by yfeng2824**
Description
- automatically run yarn fmt && git add . when committing using husky

### CODE REVIEW

File: website/.husky/pre-commit [hunk 1]
```diff
@@ -0,0 +1,3 @@
+cd website
+yarn fmt && git add .
```
Comment (RetricSu) [inline website/.husky/pre-commit:2]: does the `.husky` need to be outsite the `website` folder?

Comment (RetricSu) [inline website/.husky/pre-commit:2]: can we use https://github.com/lint-staged/lint-staged to format only the git staged files? sometimes we don't `git add` all the files

Comment (RetricSu) [reply website/.husky/pre-commit:2]: something like 
```sh
cd website
yarn lint-staged
```

and in the `package.json`, you can add:

```json
{
"scripts": {
...
"prepare": "husky install",
....
},
"lint-staged": {
    "*": [
      "prettier --write"
    ]
  },
}
```

Comment (yfeng2824) [reply website/.husky/pre-commit]: I was referring to their doc here: https://typicode.github.io/husky/how-to.html#project-not-in-git-root-directory

**Review by RetricSu — APPROVED** [review]

**Pull request merged**